src/Operators.java

```diff
@@ -1,6 +1,42 @@
 public class Operators {
-
-    public static void main (String[] args){
-
+    public static void length(String str1){
+        System.out.println("The length of the string '" + str1 + "' is: " + str1.length());
+    }
+    public static void replace(String str1, String str2, String str3){
+        System.out.println(str1.replaceAll(str2,str3));
+    }
+    public static void replacechars(String str1, String str3){
```

Comment:
This will replace spaces and . + symbols as well along with another string

Reply:
Is it not suppose to replace space, . and + sign?
I have now sorted this issue, so the method will now not replace . and + sign but I think spaces should still be removed. Please let me know if not, I will amend it.
Thanks,
Hiral